warp10/src/main/java/io/warp10/script/functions/TIMEBOX.java

```diff
@@ -155,7 +156,9 @@ public Object call() throws Exception {
       }
       throw new WarpScriptException(getName() + " reached the execution time limit (" + maxtime + " " + Constants.timeunit.name() + ").");
     } catch (ExecutionException ee) {
-      if (this.quiet && ee.getCause() instanceof WarpScriptException) {
+      if (ee.getCause() instanceof WarpScriptStopException) {
```

Comment:
Instead of WarpScriptStopException, shouldn't WarpScriptATCException be handled?

Reply:
```
<% 
 BREAK
%> 10000 TIMEBOX
```
must still fail.

Reply:
WarpScriptKillException still remains a good question, but it is not used outside the UDP plugin.